oauthdoctor/oauth/helper.go

```diff
@@ -255,17 +254,23 @@ func (c *Config) oauth2Client(code string) (*http.Client, string) {
 	return conf.Client(oauth2.NoContext, token), token.RefreshToken
 }
 
+var apiURL = "https://googleads.googleapis.com/v1/customers/"
```

Comment:
Why is there a default value if you are going to override it in the tests?

Reply:
The default value is needed for the actual Google Ads API call. It is declared here, so the unit test can override the URL from httptest server.